aptos-move/block-executor/src/task.rs

```diff
@@ -28,30 +24,6 @@ pub enum ExecutionStatus<O, E> {
     SkipRest(O),
 }
 
-/// Trait that defines a transaction type that can be executed by the block executor. A transaction
-/// transaction will write to a key value storage as their side effect.
-pub trait Transaction: Sync + Send + Clone + 'static {
```

Comment:
I understand that you've renamed it because you needed the type externally, and need to distinguish it. 
But this is going to be a huge conflict with an aggregator branch.

how about having an alias for this module :
// TODO rename to BlockExecutableTransaction after aggregators get merged
type Transaction = BlockExecutableTransaction;

and undoing all the changes in block executor with regards to that?

Reply:
Done.